### ISSUE TITLE
in PROD: in Requests tab, BorrowDirect items still show "SUL" and have SW link

### ISSUE THREAD

**Issue by saseestone** [sul-dlss/mylibrary#434]
Can't tell if this is a prod thing and it's fixed in dev...but the test BD request I made is now on campus and ready for me to pickup.  The details shows the item at Source: SUL and the SW link is showing.

Checkouts tab is all good, but Requests tab needs some help.

![image](https://user-images.githubusercontent.com/14897546/62404234-8a5a7400-b547-11e9-97cb-4edb0441b85d.png)

**Comment by camillevilla** [comment]
@saseestone we're 🤞 that this was resolved by a deploy this afternoon. Let us know otherwise.

**Comment by cbeer** [comment]
In #394, we explicitly did only checkouts. @jkeck said "when an ON_ORDER item is requested its library is SUL", so maybe we need a better way to differentiate borrowdirect requests on the requests tab?

**Comment by saseestone** [comment]
Ah yes, of course we did. (Meaning - you did check with me and I did point us to do it this way...) 

But I've had 2 different staff members point out the SW link on a BorrowDirect item they have requested and note that they are from "SUL". (And on the first day of staff testing.)  

I checked in with Darsi on the order process (something I definitely should have done earlier...), and she tells me that most on-order items don't have library = SUL.  That Acq knows the holding code at the point of order, and the order is created with the correct library.  

I ran a sym report and the number of items with Curr Loc = ON-ORDER and Library = SUL is 35.  (And Darsi is confident these are mistakes.)  I also went to SW,  clicked on the "On order" facet, and paged through the items for a bit.  All had libraries.

So I'm confident that we can apply the translation code for the Checkouts tab to Requests.  Sorry to not do the due diligence earlier.
